compliance/locker.py

```diff
@@ -41,7 +41,7 @@
 INDEX_FILE = 'index.json'
 DAY = 60 * 60 * 24
 AE_DEFAULT = 30 * DAY
-AE_EXEMPT = [INDEX_FILE, 'README.md', 'readme.md']
+AE_EXEMPT = [INDEX_FILE, 'README.md', 'readme.md', 'Readme.md']
```

Comment:
What about mkaing this case insensative vi `re`?

Reply:
I thought about that but I decided against it to guard against exempting files like 'rEaDmE.mD`.